core/templates/dev/head/pages/base.html

```diff
@@ -225,7 +225,6 @@
 
     <script src="{{TEMPLATE_DIR_PREFIX}}/domain/sidebar/SidebarStatusService.js"></script>
     <script src="{{TEMPLATE_DIR_PREFIX}}/domain/utilities/UrlInterpolationService.js"></script>
-
```

Comment:
Please could you check the diff [here](https://github.com/oppia/oppia/pull/3669/files) and remove all the superfluous edits to files? They distract from the main changes you're trying to make.